src/backend/src/modules/gcnotify/gcnotify.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { KeycloakUser } from '../auth/keycloak-user.model';
+import { GcNotifyEmailDto } from './dto/gcnotify-pia.dto';
+
+@Injectable()
+export class GcNotifyService {
+  constructor(private readonly httpService: HttpService) {}
+
+  async sendEmail(
+    user: KeycloakUser,
+    mpoEmail: string,
+    pia: GcNotifyEmailDto,
```

Comment:
I think the HTTP service uses the observable pattern, if you want to convert to Promise, you should use firstValueFrom from rsjx. You can find it from Auth module